README.md

```diff
@@ -87,6 +88,14 @@ mars5, config_class = torch.hub.load('Camb-ai/mars5-tts', 'mars5_english', trust
 # The `mars5` contains the AR and NAR model, as well as inference code.
 # The `config_class` contains tunable inference config settings like temperature.
 ```
+
```

Comment:
Add instruction to clone the repo before doing this, otherwise this code block will not work.